llama-index-integrations/llms/llama-index-llms-fireworks/llama_index/llms/fireworks/utils.py

```diff
@@ -0,0 +1,114 @@
+from typing import Any, Dict, List, Optional, Sequence, Tuple
+
+from llama_index.core.base.llms.types import ChatMessage, MessageRole
+from llama_index.core.llms.generic_utils import get_from_param_or_env
```

Comment:
Should be `from llama_index.core.base.llms.generic_utils import get_from_param_or_env`